### PR TITLE
Higher level libraries better description

### PR DESCRIPTION
**Pull request by mensfeld**
I've read today some docs, and noticed, that the readme is getting a bit confusion on this one, so updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -986,16 +988,22 @@ Latest stable release, with native support for the Kafka 0.10 protocol and event
...
 
 * [Karafka](https://github.com/karafka/karafka) - Framework used to simplify Apache Kafka based Ruby and Rails applications development. Karafka provides higher abstraction layers, including Capistrano, Docker and Heroku support.
 
 * [Phobos](https://github.com/klarna/phobos) - Micro framework and library for applications dealing with Apache Kafka. It wraps common behaviors needed by consumers and producers in an easy and convenient API.
 
+### Messages publishing libraries
+
+* [DeliveryBoy](https://github.com/zendesk/delivery_boy) – A library that integrates with Ruby on Rails, making it easy to publish Kafka messages from any Rails application.
+
+* [WaterDrop](https://github.com/zendesk/delivery_boy) – A library for Ruby and Ruby on Rails applications, to easy publish Kafka messages in both sync and async way.
```
Comment (Elyahou) [inline README.md]: the url is not the good one

Comment (mensfeld) [reply README.md]: true. Will fix in a sec.

File: README.md [hunk 2]
```diff
@@ -986,16 +988,22 @@ Latest stable release, with native support for the Kafka 0.10 protocol and event
...
+### Frameworks
 
-* [DeliveryBoy](https://github.com/zendesk/delivery_boy) – A library that integrates with Ruby on Rails, making it easy to publish Kafka messages from any Rails application.
+* [Racecar](https://github.com/zendesk/racecar) - A simple framework that integrates with Ruby on Rails to provide a seamless way to write, test, configure, and run Kafka consumers. It comes with sensible defaults and conventions.
 
 * [Karafka](https://github.com/karafka/karafka) - Framework used to simplify Apache Kafka based Ruby and Rails applications development. Karafka provides higher abstraction layers, including Capistrano, Docker and Heroku support.
 
 * [Phobos](https://github.com/klarna/phobos) - Micro framework and library for applications dealing with Apache Kafka. It wraps common behaviors needed by consumers and producers in an easy and convenient API.
 
+### Messages publishing libraries
```
Comment (dasch) [inline README.md]: `s/Messages/Message/`

File: README.md [hunk 3]
```diff
@@ -986,16 +988,22 @@ Latest stable release, with native support for the Kafka 0.10 protocol and event
 
 ## Higher level libraries
 
-Currently, there are three actively developed frameworks based on ruby-kafka, that provide higher level API that can be used to work with Kafka messages:
+Currently, there are three actively developed frameworks based on ruby-kafka, that provide higher level API that can be used to work with Kafka messages and two libraries for publishing messages.
 
-* [Racecar](https://github.com/zendesk/racecar) - A simple framework that integrates with Ruby on Rails to provide a seamless way to write, test, configure, and run Kafka consumers. It comes with sensible defaults and conventions.
+### Frameworks
```
Comment (dasch) [inline README.md]: This might better be described as _message processing frameworks_.

Comment (mensfeld) [reply README.md]: not quite sure about that though as karafka is also a message producing framework and there's an issue for that in racecar as well

Comment (dasch) [reply README.md]: I think _processing_ typically includes producing messages, as well – for instance, Kafka Streams is described as _stream processing_.

Comment (mensfeld) [reply README.md]: @dasch  good point. Will update accordingly.

**Comment by dasch** [comment]
Thanks!

**Pull request merged**